gcp/observability/opencensus.go

```diff
@@ -118,3 +121,22 @@ func startOpenCensus(config *config) error {
 
 	return nil
 }
+
+// stopOpenCensus flushes the exporter's and cleans up globals across all
+// packages if exporter was created.
+func stopOpenCensus() {
+	if exporter != nil {
+		internal.ClearExtraDialOptions()
+		internal.ClearExtraServerOptions()
+
+		// Call these unconditionally, doesn't matter if not registered, will be
+		// a noop if not registered.
+		trace.UnregisterExporter(exporter)
+		view.UnregisterExporter(exporter)
+
+		if sdExporter, ok := exporter.(*stackdriver.Exporter); ok {
```

Comment:
Why not define `Flush` and `Close` methods on `tracingMetricsExporter` instead?  Does it make the tests unnecessarily complex?

Reply:
Done.